### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
-  Cleaning up

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: scripts/install [hunk 1]
```diff
@@ -109,13 +108,15 @@ chmod -R 755 "$final_path/plugins/Morpheus"
...
 ynh_script_progression --message="Setuping a cron..."
 
-cp ../conf/cron /etc/cron.d/$app
+# cp ../conf/cron /etc/cron.d/$app
+
+# ynh_replace_string --match_string="__FINALPATH__" --replace_string="$final_path" --target_file="/etc/cron.d/$app"
+# ynh_replace_string --match_string="__APP__" --replace_string="$app" --target_file="/etc/cron.d/$app"
+# ynh_replace_string --match_string="__DOMAIN__" --replace_string="$domain" --target_file="/etc/cron.d/$app"
+# ynh_replace_string --match_string="__PATH__" --replace_string="$path_url" --target_file="/etc/cron.d/$app"
+# ynh_replace_string --match_string="__PHPVERSION__" --replace_string="$phpversion" --target_file="/etc/cron.d/$app"
```
Comment (alexAubin) [inline scripts/install:117]:
```suggestion
```

Let's remove these if we ain't using them

File: scripts/upgrade [hunk 2]
```diff
@@ -169,13 +159,15 @@ fi
...
 ynh_script_progression --message="Setuping a cron..."
 
-cp ../conf/cron /etc/cron.d/$app
+# cp ../conf/cron /etc/cron.d/$app
+
+# ynh_replace_string --match_string="__FINALPATH__" --replace_string="$final_path" --target_file="/etc/cron.d/$app"
+# ynh_replace_string --match_string="__APP__" --replace_string="$app" --target_file="/etc/cron.d/$app"
+# ynh_replace_string --match_string="__DOMAIN__" --replace_string="$domain" --target_file="/etc/cron.d/$app"
+# ynh_replace_string --match_string="__PATH__" --replace_string="$path_url" --target_file="/etc/cron.d/$app"
+# ynh_replace_string --match_string="__PHPVERSION__" --replace_string="$phpversion" --target_file="/etc/cron.d/$app"
```
Comment (alexAubin) [inline scripts/upgrade:168]:
```suggestion
```

Let's remove these if we ain't using them

**Review by alexAubin — APPROVED** [review]
LGTM

**Comment by yalh76** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Pull request merged**